LICENSE

```diff
@@ -1,6 +1,8 @@
 MIT License
 
-Copyright (c) 2023 Lawrence Livermore National Laboratory
+See the COPYRIGHT file for ownership and authorship information.
```

Comment:
Shouldn't you put that in every file it would be easier if we update the copyright in the future?

Reply:
I'm confused on what you mean here. Every file has the following statement in a copyright header: "See top-level LICENSE and COPYRIGHT files for dates and other details." Are you talking about using the exact wording listed above?

Reply:
I meant wherever you have the full copyright text, why not point to the file as you do here